config/services.php

```diff
@@ -23,6 +23,14 @@
         'secret' => env('MANDRILL_SECRET'),
     ],
 
+    'northstar' => [
+        'grant' => 'password',
+        'url' => env('NORTHSTAR_URL'),
+        'client_id' => env('NORTHSTAR_CLIENT_ID'),
+        'client_secret' => ent('NORTHSTAR_CLIENT_SECRET'),
+        'scope' => ['user', 'admin'],
```

Comment:
Yep, so these determine what kind of abilities the access token that Northstar grants will have. The `user`, `role:staff`, and `role:admin` will allow the token to act with as much authority as the user has on their account (so like `role:admin` allows you to act as an admin, but only if your Northstar user has the admin role).

The `admin` scope grants privileges and is more intended for "machine" clients that wouldn't have a role attached to them (yet! it's probably something we should change).